enterprise/models.py

```diff
@@ -1366,6 +1377,34 @@ def __repr__(self):
         return self.__str__()
 
 
+@python_2_unicode_compatible
+class LicensedEnterpriseCourseEnrollment(TimeStampedModel):
+    """
+    An Enterprise Course Enrollment that is enrolled via a license.
+
+    .. no_pii:
+    """
+
+    license_uuid = models.UUIDField(
+        primary_key=False,
+        editable=False,
+        null=False
+    )
+
+    enterprise_course_enrollment = models.OneToOneField(
+        EnterpriseCourseEnrollment,
+        blank=False,
+        null=True,
```

Comment:
I think we want `null=False` here, too.  You shouldn't be able to create one of these without associating it with an existing `EnterpriseCourseEnrollment`.